packages/non-deployable/token-allocation/README.md

```diff
@@ -8,21 +8,22 @@ The adapter takes the following environment variables:
-|           | `COINMETRICS_ADAPTER_URL  ` |  The location of a CoinMetrics external adapter  |         |             |
-|           |  `COINPAPRIKA_ADAPTER_URL`  |  The location of a CoinPaprika external adapter  |         |             |
-|           |  `COINRANKING_ADAPTER_URL`  |  The location of a CoinRanking external adapter  |         |             |
-|           | `CRYPTOCOMPARE_ADAPTER_URL` | The location of a CryptoCompare external adapter |         |             |
-|           |    `FINAGE_ADAPTER_URL`     |    The location of a Finage external adapter     |         |             |
-|           |     `KAIKO_ADAPTER_URL`     |     The location of a Kaiko external adapter     |         |             |
-|           |    `NCFX_ADAPTER_URL  `     |     The location of a NCFX external adapter      |         |             |
-|           |    `TIINGO_ADAPTER_URL`     |    The location of a Tiingo external adapter     |         |             |
+| Required? |              Name               |                     Description                     | Options | Defaults to |
+| :-------: | :-----------------------------: | :-------------------------------------------------: | :-----: | :---------: |
+|           |     `AMBERDATA_ADAPTER_URL`     |    The location of an Amberdata external adapter    |         |             |
+|           |   `CFBENCHMARKS_ADAPTER_URL`    |   The location of a CFBenchmarks external adapter   |         |             |
+|           |      `COINAPI_ADAPTER_URL`      |     The location of a CoinAPI external adapter      |         |             |
+|           |     `COINGECKO_ADAPTER_URL`     |    The location of a CoinGecko external adapter     |         |             |
+|           |   `COINMARKETCAP_ADAPTER_URL`   |  The location of a CoinMarketCap external adapter   |         |             |
+|           |   `COINMETRICS_ADAPTER_URL  `   |   The location of a CoinMetrics external adapter    |         |             |
+|           |    `COINPAPRIKA_ADAPTER_URL`    |   The location of a CoinPaprika external adapter    |         |             |
+|           |    `COINRANKING_ADAPTER_URL`    |   The location of a CoinRanking external adapter    |         |             |
+|           |   `CRYPTOCOMPARE_ADAPTER_URL`   |  The location of a CryptoCompare external adapter   |         |             |
+|           |      `FINAGE_ADAPTER_URL`       |      The location of a Finage external adapter      |         |             |
+|           |       `KAIKO_ADAPTER_URL`       |      The location of a Kaiko external adapter       |         |             |
+|           |       `NCFX_ADAPTER_URL`        |       The location of a NCFX external adapter       |         |             |
+|           |      `TIINGO_ADAPTER_URL`       |      The location of a Tiingo external adapter      |         |             |
+|           | `BLOCKSIZE_CAPITAL_ADAPTER_URL` | The location of a BlocksizeCapital external adapter |         |             |
```

Comment:
Small nit, would you be able to alphabetize the new lines in the README?

Reply:
Done!